### DISCUSSION TITLE
Does quickview still work

### DISCUSSION THREAD

**Discussion by Blobby-Boi** [3kh0/ext-remover#975] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I was looking through the list of exploits and I saw one called quickview, does it still work?

**Comment by AshtonDavies** [comment]
Bookmarklets are patched for the typical policy setup.

**Reply by Blobby-Boi** [reply]
Is there anyway to get them unpatched (other than the google lens thing that is still not done)?

**Reply by AshtonDavies** [reply]
> Is there anyway to get them unpatched (other than the google lens thing that is still not done)?

Only other way is with extensions.

**You have two options:**
- uBlock Origin execution
- Snap&Read execution

**Reply by Blobby-Boi** [reply]
I have both pre-installed lol

**Comment by S-PScripts** [comment, marked as answer]
qukcgnn

**Reply by Blobby-Boi** [reply]
💀
